CotRotC.md

```diff
@@ -611,6 +611,12 @@ The Secretary-General of the United Nations is designated as the depositary of t
 
 The original of the present Convention, of which the Arabic, Chinese, English, French, Russian and Spanish texts are equally authentic, shall be deposited with the Secretary-General of the United Nations. In witness thereof the undersigned plenipotentiaries, being duly authorized thereto by their respective Governments, have signed the present Convention.
 
+### Article 54
+
+Free candy for all. 
```

Comment:
While I understand the intent of the article it must be noticed that the wording does not honor the previous work done by the sponsors of this Chart.

Reply:
What about?
"Under the auspicious of universality we deem necessary for those covered by this Chart to have the right of free candy and other indulgences not only on Sundays or any other significant holiday".